File: LeetCode/cpp/217.cpp
class Solution {
public:
    bool containsDuplicate(vector<int> &nums) {
        unordered_set<int> table;
        for (auto n: nums) {
            if (table.find(n) == table.end())
                table.insert(n);
            else
                return true;
        }
        return false;
    }
};

File: LeetCode/python3/217.py
class Solution:
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        hash_table = set()
        for i in nums:
            if i in hash_table:
                return True
            else:
                hash_table.add(i)
        return False